tests/transforms/test_metric_tensor.py

```diff
@@ -207,7 +209,8 @@ def test_evaluate_diag_metric_tensor(self, tol):
         block-diagonal and diagonal setting."""
         dev = qml.device("default.qubit", wires=2)
 
-        def circuit(a, b, c):
+        def circuit(abc):
+            a, b, c = abc

```

Comment:
Why was this changed to a single argument?

Reply:
With the new behaviour, the metric tensor would be returned per QNode argument, yielding a tuple of 3 `(1, 1)` arrays, and in particular only containing the information about the _diagonal_ metric entries.
By making it a single QNode argument, the full tensor is returned and we have more entries to test the code on :)